src/notebooks/ingest_loop_for_segfaults.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Script to brute-force succeed with data ingest despite sporadic segmentation
+# faults with Kuzu data ingest especially exhibited during certain complex
+# (many distinct node pair types) REL tables. Intended to be used manually.
```

Comment:
I recall that you raised an issue with the Kuzu devs about this, possibly https://github.com/kuzudb/kuzu/issues/2920? It might be good to link it here so that future users can stay on top of possible fixes.

Reply:
Thanks! It took me some time after this PR was opened to create kuzudb/kuzu#2920 with a less complex demonstration (so it wasn't originally mentioned here). I'll add it in as a reference to help readers understand that this might yet eventually be fixed (and there are no data transaction worries).